lib/env-bundler.js

```diff
@@ -930,7 +930,7 @@ async function generateBundle (options, components) {
   }));
 
   // Filter unique resources and save into resources folder
-  const uniqueUsedFiles = resources.reduce((a, b) => a.concat(b.filter(i => !a.includes(i))), []);
+  const uniqueUsedFiles = resources.flat().filter(onlyUnique);
   await saveResources(path, uniqueUsedFiles);
```

Comment:
This fix is about the filtering function. It's not a problem but I found it during the debug session. The !a.includes(i) couldn't check for duplicates because a is reassigned at the end of the process.